cmd/feedback/feedback.go

```diff
@@ -242,6 +243,12 @@ func runFeedbackCommand(ctx context.Context, clients *shared.ClientFactory, cmd
 		return nil
 	}
 
+	// DEPRECATED(semver:major): Support the deprecated survey name for backwards compatibility
+	if surveyNameFlag == SlackPlatformFeedbackDeprecated {
+		surveyNameFlag = SlackPlatformFeedback
+		clients.IO.PrintDebug(ctx, "DEPRECATED: The '--name %s' flag is deprecated; use '--name %s' instead", SlackPlatformFeedbackDeprecated, SlackPlatformFeedback)
+	}
+
```

Comment:
note: I print a deprecation message to our `PrintDebug` instead of `PrintWarning`. We use both. Preference?

Reply:
📣 I'm in favor of annoying and loud deprecation messages to encourage making expected changes as soon as possible.

This is a pattern I've found in build tools like `goreleaser` and `nix` that can be frustrating at first, but also save headache when the actual breaking changes do happen!

Reply:
Cool with me! Commit 06e247e now prints this loud and in the open!

<img width="762" alt="image" src="https://github.com/user-attachments/assets/358a85b2-4919-4019-80b3-1662e9ece5a5" />